generators/client/templates/angular/src/main/webapp/app/admin/metrics/metrics-modal-threads/metrics-modal-threads.component.html.ejs

```diff
@@ -17,85 +17,85 @@
  limitations under the License.
 -%>
 <div class="modal-header">
-    <h4 class="modal-title" <%= jhiPrefix %>Translate="metrics.jvm.threads.dump.title">Threads dump</h4>
+  <h4 class="modal-title" <%= jhiPrefix %>Translate="metrics.jvm.threads.dump.title">Threads dump</h4>
```

Comment:
Currently we have 4 space spacing in HTML files. We should follow the same style.  
If we want to go for 2 space spacing in HTML files then this should be done systematically for all HTML files.

Reply:
fix